src/powerman/device.c

```diff
@@ -680,7 +685,7 @@ static int _enqueue_targeted_actions(Device * dev, int com, hostlist_t hl,
             }
 
             act = _create_action(dev, com, plugs, complete_fun, vpf_fun,
-                        client_id, arglist);
+                                 dpf_fun, client_id, arglist);
```

Comment:
whitespace issue there?

Reply:
in this case intentional.  Powerman uses both

```
function(a, b, c, d
  e, f, g);
```

and

```
function(a, b, c, d
              e, f, g);
```

styles.  I think the latter is the more common approach today.  So in the few circumstances that a change half-forced a change from the former style to the latter, i made the change.